### ISSUE TITLE
Module not found error: No module named 'flee', installing Flee in ubuntu 

### ISSUE THREAD

**Issue by yserrano-ds** [djgroen/flee#40]
Hi, please excuse me, this is probably a very basic question.
I am installing Flee in Ubuntu 22.10 for testing purposes.  It is a VirtualBox VM. I created and activated a virtualenv for this installation. I followed the steps of [installing/testing flee](https://flee.readthedocs.io/en/latest/Installation_and_Testing/#testing-flee). But in the last step I got this error:

$ python3 tests/test_csv.py
Traceback (most recent call last):
  File "...flee/tests/test_csv.py", line 4, in <module>
    import flee.postprocessing.analysis as a
ModuleNotFoundError: No module named 'flee'

Thank you,

Yonny

**Comment by djgroen** [comment]
Hi Yonny,

This documentation is slightly outdated, but basically you need to add your main flee installation directory to your PYTHONPATH environment variable, e.g. by including it in .bashrc if you use bash.

Does that help to resolve your issue?

**Comment by yserrano-ds** [comment]
Yes, it worked.  I had to resolve other dependencies (pandas, numpy and yaml) and after that the test ran successfully.
Thank you!
